.github/workflows/bootstrap.yml

```diff
@@ -3,19 +3,10 @@ name: Bootstrapping
 on:
   # This Workflow can be triggered manually
   workflow_dispatch:
-  pull_request:
-    branches:
-      - develop
-      - releases/**
-    paths-ignore:
-      # Don't run if we only modified packages in the
-      # built-in repository or documentation
-      - 'var/spack/repos/builtin/**'
-      - '!var/spack/repos/builtin/packages/clingo-bootstrap/**'
-      - '!var/spack/repos/builtin/packages/clingo/**'
-      - '!var/spack/repos/builtin/packages/python/**'
-      - '!var/spack/repos/builtin/packages/re2c/**'
-      - 'lib/spack/docs/**'
+  workflow_run:
+    workflows: [ "style" ]
```

Comment:
Should there be branches-ignore here too, or do we want to re-run on all merges?

Reply:
They don't compose, but because the style one only runs on the normal branch checks, and this only gets triggered when style finishes and succeeds, it should only trigger on those branches.  Worth testing though for sure.

Reply:
For some reason the workflows gated after style didn't run.

Reply:
I’ve been looking at that, apparently this kind of trigger only works once it exists on the develop or main branch. Will need to make up a test repo to the side to debug this. May be better to rework it as a reusable action and invoke from style instead.